tests/Benchmarks/System.Text.Utf8/Utf8StringPerf.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Text;
+using System.Text.Utf8;
+using BenchmarkDotNet.Attributes;
+
+namespace Benchmarks.System.Text.Utf8
```

Comment:
We decided on the namespace structure (like unit tests) as `<namespace>.Benchmarks`